test/abstractProjectCreateCommand.unit.test.ts

```diff
@@ -127,19 +134,66 @@ describe('abstractProjectCreateCommand command', () => {
         },
         {
           dbDialect: 'postgres',
-          agent: 'express-sequelize',
+          agent: Agents.ExpressSequelize,
           isLocal: true,
           architecture: 'microservice',
           projectId: 1,
         },
       );
     });
+
+    it('should throw an error when there is a 401 error', async () => {
+      expect.assertions(1);
+
+      const { stubs, instance } = setup();
+
+      const error = {
+        status: 401,
+      };
+
+      stubs.spinner.attachToPromise.mockRejectedValue(error);
+
+      await expect(instance.run()).rejects.toBe(error);
+    });
+
+    it('should throw an error when there is a 403 error', async () => {
+      expect.assertions(1);
+
+      const { stubs, instance } = setup();
+
+      const error = {
+        status: 403,
+      };
+
+      stubs.spinner.attachToPromise.mockRejectedValue(error);
+
+      await expect(instance.run()).rejects.toBe(error);
+    });
+
+    it('should exit with code 1 when the error is not a 401 or a 403', async () => {
```

Comment:
🤮 Ok this is not your fault, this has been coded like that. However, erreur management should reaaaaally be reworked. Commands should never exit on error they were not supposed to catch... Errors should be bubble up upper, in the global error handle in OCLIF bin/run